Monika After Story/game/definitions.rpy

```diff
@@ -6732,7 +6732,13 @@ init python:
         """
         if check_time is None:
             check_time = datetime.datetime.now()
-        return persistent._mas_last_monika_ily is not None and (check_time - persistent._mas_last_monika_ily) <= pass_time
+
+        # if a backward TT is detected here, return False and reset persistent._mas_last_monika_ily
+        if persistent._mas_last_monika_ily is None or persistent._mas_last_monika_ily > check_time:
+            persistent._mas_last_monika_ily = None
```

Comment:
do we call this function with a value passed into `check_time` anywhere? 

The `check_time` parameter doesn't make sense with this change since you could purposefully pass in an old `check_time` and change game state.
If it's unused, we should dummy out the parameter.
If it's lightly used, then we should make a new function for `check_time` usage and replace the appropriate lines.
If it's heavily used, then we should consider a new function that doesn't use `check_time` at all.

Reply:
this func is only used once, and that's when we load the Talk menu to check for this one purpose. It honestly doesn't need check_time as a passed in param, just as an internal var that is set to now